aten/src/ATen/native/MathBitsFallback.cpp

```diff
@@ -0,0 +1,119 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <torch/library.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <ATen/native/UnaryOps.h>
+#include <ATen/NativeFunctions.h>
+
+namespace at {
+
+struct MathOpFallback {
```

Comment:
Strongly recommend leaving a comment at the top of the class giving a general idea about what this abstracts over and how to use the abstraction.

Reply:
done!